packages/paste-core/primitives/box/src/fakeCss.js

```diff
@@ -0,0 +1,217 @@
+// based on https://github.com/developit/dlv
+export const get = (obj, key, def, p, undef) => {
+  key = key && key.split ? key.split('.') : [key];
+  for (p = 0; p < key.length; p++) {
+    obj = obj ? obj[key[p]] : undef;
+  }
+  return obj === undef ? def : obj;
+};
+
+const defaultBreakpoints = [40, 52, 64].map(n => n + 'em');
+
+const defaultTheme = {
+  space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
+  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64, 72],
+};
+
+// Do we want these?
+const aliases = {
+  bg: 'backgroundColor',
+  m: 'margin',
+  mt: 'marginTop',
+  mr: 'marginRight',
+  mb: 'marginBottom',
+  ml: 'marginLeft',
+  mx: 'marginX',
+  my: 'marginY',
+  p: 'padding',
+  pt: 'paddingTop',
+  pr: 'paddingRight',
+  pb: 'paddingBottom',
+  pl: 'paddingLeft',
+  px: 'paddingX',
+  py: 'paddingY',
+};
+
+const multiples = {
+  marginX: ['marginLeft', 'marginRight'],
+  marginY: ['marginTop', 'marginBottom'],
+  paddingX: ['paddingLeft', 'paddingRight'],
+  paddingY: ['paddingTop', 'paddingBottom'],
+  size: ['width', 'height'],
+};
+
+const scales = {
+  color: 'colors',
+  backgroundColor: 'backgroundColors',
+  borderColor: 'borderColors',
```

Comment:
I had to change what these mapped to from the normal theme keys in styled-system https://styled-system.com/css/#theme-keys because we differ.  Not sure if anything else needed changing.